### PR TITLE
[19.03] added entry to changelog

### PR DESCRIPTION
**Pull request by andrewhsu**
added entry to changelog

### CODE REVIEW

**Comment by thaJeztah** [comment]
Do we actually need this? Given that 19.03 > 18.09.8, so it's not a change since 18.09

File: CHANGELOG.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ For official release notes for Docker Engine CE and Docker Engine EE, visit the
 - Fix: fetch the right device number when greater than 255 and using `--device-read-bps` option. [moby/moby#39212](https://github.com/moby/moby/pull/39212)
 - Fix: "Path does not exist" error when path definitely exists. [moby/moby#39251](https://github.com/moby/moby/pull/39251)
 - Fix: [CVE-2018-15664](https://nvd.nist.gov/vuln/detail/CVE-2018-15664) symlink-exchange attack with directory traversal. [moby/moby#39357](https://github.com/moby/moby/pull/39357)
```
Comment (thaJeztah) [inline CHANGELOG.md]: We might actually remove this one as well, because it was already released in 18.09.x

Comment (andrewhsu) [reply CHANGELOG.md]: I think this is still representative of what was brought into the 19.03 branch for the GA release.

**Comment by andrewhsu** [comment]
rebased

**Pull request merged**